app/views/configure/display.phtml

```diff
@@ -71,8 +71,18 @@
 								<?php } ?>
 							</div>
 							<div class="properties">
-								<div><?= sprintf('%s — %s %s', $theme['name'], _t('gen.short.by_author'), $theme['author']) ?></div>
-								<div><?= $theme['description'] ?></div>
+								<div>
+									<?php if (isset($theme['deprecated']) && $theme['deprecated']) { ?>
+										<span class="deprecated error"><?= _t('conf.display.theme.deprecated') ?>:</span>
+									<?php } ?>
+									<?= sprintf('%s — %s %s', $theme['name'], _t('gen.short.by_author'), $theme['author']) ?>
+								</div>
+								<div>
+									<?php if (isset($theme['deprecated']) && $theme['deprecated']) { ?>
+										<span class="deprecated"><?= _t('conf.display.theme.deprecated.description') ?></span><br />
+									<?php } ?>
+									<?= $theme['description'] ?>
```

Comment:
Have we checked that this is already HTML-encoded?

Reply:
It looks fine:

![grafik](https://user-images.githubusercontent.com/1645099/211668657-a1a0d5c0-6bd5-4b76-b89c-9e188371e077.png)

![grafik](https://user-images.githubusercontent.com/1645099/211669072-2b3930f2-be94-4ff3-8440-57da3231bf5c.png)